src/cogs/split.py

```diff
@@ -0,0 +1,203 @@
+    async def lobby(self, ctx: Context, lobby: VoiceChannel) -> None:
+        """Set the lobby channel to be used for splitting."""
+        self.data.lobby_channel_id = lobby.id
+        await ctx.send(f"> Lobby set to {lobby.mention}", delete_after=5)
+
+    @config.command(aliases=["one", "team1", "t1"])
+    async def team_one(self, ctx: Context, channel: VoiceChannel) -> None:
+        """Set the voice channel for team one."""
+        self.data.team_one_channel_id = channel.id
+        await ctx.send(f"> Team One set to {channel.mention}", delete_after=5)
+
+    @config.command(aliases=["two", "team2", "t2"])
+    async def team_two(self, ctx: Context, channel: VoiceChannel) -> None:
+        """Set the voice channel for team two."""
+        self.data.team_two_channel_id = channel.id
+        await ctx.send(f"> Team Two set to {channel.mention}", delete_after=5)
+
+    @lobby.error
+    @team_two.error
+    @team_one.error
+    async def channel_error(self, ctx: Context, error: CommandError) -> None:
+        if isinstance(error, BadArgument):
+            await ctx.send("> Please make sure you are tagging a **Voice Channel**.")
+
+    @config.command(aliases=["min_size", "min"])
+    async def minimum_team_size(self, ctx: Context, min_team_size: int) -> None:
+        """Set the minimum players per team."""
+        if min_team_size <= 0:
+            await ctx.send("> Idk chief, this seems kinda weird. (*Please chose a number greater than 0*()", delete_after=5)
+            return
+
+        self.data.minimum_team_size = min_team_size
+        await ctx.send(f"> Minimum team size set to {min_team_size}", delete_after=5)
+
+    @config.command(aliases=["cap", "leader", "role"])
+    async def captain_role(self, ctx: Context, role: Role) -> None:
+        """Set the captain role."""
+        self.data.captain_role_id = role.id
+        await ctx.send(f"> Set captain role to {role.mention}", delete_after=5, allowed_mentions=AllowedMentions(roles=False))
+
+    @command()
+    async def split(self, ctx: Context) -> None:
+
+        # Get a list of invalid settings and display them to the user if there is
+        # at least one invalid setting before trying to split.
+        invalid_settings = []
+        if self.lobby_channel is None:
+            invalid_settings.append("Lobby")
+        if self.team_one_channel is None:
+            invalid_settings.append("Team One")
+        if self.team_two_channel is None:
+            invalid_settings.append("Team Two")
+
+        if len(invalid_settings) > 0:
+            await ctx.send(f"> You must configure the following settings before a split: {', '.join(invalid_settings)}", delete_after=5)
+            return
+
+        # FIXME: Find the underlying api request to get this info.
+        # This list is generated from the discord.py cache.
+        # This means if the user has been in the channel since the bot has come online,
+        # They will not be inside this list.
```

Comment:
It is rarely appropriate to leave "TODO" or "FIXME" comments in the final code. We have a ticketing system on github. These should be created as issues if they are important enough to need to be implemented. Leaving a todo comment encourages it to be skipped over and forgotten.

Reply:
Understandable, an issue has been created. https://github.com/fisher60/spliteam/issues/5